compiler/q-implant/README.md

```diff
@@ -2,13 +2,21 @@
 
 _q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
 
+Therefore, before proceeding with q-implant, you should define `qparam.json` and `\*.npy` files so that they can be applied to the same operand as the circle you want to apply.
```

Comment:
1. Please add format (or content) of this `qparam.json` so that anyone can provide this.
2. Just `.npy` is not specific. It would be better to describe how to provide this file, required condition of this file, relation with the target circle model.